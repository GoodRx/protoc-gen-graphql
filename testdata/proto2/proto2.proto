syntax = "proto2";

package protoc_gen_graphql.test.proto2;

message Test {
  optional string optional_field = 1;
  required string required_field = 2;
  repeated string repeated_field = 3;
}
