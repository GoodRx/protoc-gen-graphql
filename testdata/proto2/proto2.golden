# DO NOT EDIT! Generated by protoc-gen-graphql.

type ProtocGenGraphqlTestProto2_Test {
  optionalField: String
  requiredField: String!
  repeatedField: [String!]!
}
